lib/metrics/amplitude.js

```diff
@@ -165,23 +173,56 @@ module.exports = log => {
 
     return P.resolve()
   }
-}
 
-function getFromToken (request, key) {
-  if (request.auth.credentials) {
-    return request.auth.credentials[key]
+  function getFromToken (request, key) {
+    if (request.auth.credentials) {
+      return request.auth.credentials[key]
+    }
   }
-}
```

Comment:
I moved all of these functions inside the closure to keep them at the same level as `mapUserProperties`.